src/core/stdc/stdarg.d

```diff
@@ -34,120 +85,183 @@ unittest
     assert((-9).alignUp!8 == -8);
 }
 
-version (X86)
+
+/**
+ * The argument pointer type.
+ */
+version (GNU)
+{
+    alias va_list = __gnuc_va_list;
+    alias __gnuc_va_list = __builtin_va_list;
```

Comment:
I know order of evaluation doesn't matter, but swap the two around?

Reply:
It's for the ddoc comment (one for all version branches).

Reply:
Ah, ok then.